### PR TITLE
[1LP][RFR] Updated is_diplayed of AutomateImportExportView

### PR DESCRIPTION
**Pull request by ganeshhubale**
- Fixed reset button xpath
- Added ```reset_title``` widget to read title

{{ pytest: cfme/tests/automate/test_git_import.py -k 'test_automate_git_domain_removed_from_disk' -vv }}

### CODE REVIEW

File: cfme/base/ui.py [hunk 1]
```diff
@@ -1709,8 +1709,9 @@ class import_git(View):     # noqa
     reset_all = VersionPick({
         Version.lowest(): Image(
             './/img[starts-with(@alt, "Reset all components in the following domains:")]'),
-        '5.10': Button(title='Reset all components in the following domains:')
+        '5.10': Button(title='Reset all components in the following domains: RedHat, ManageIQ')
     })
+    reset_title = Text('//*[@id="main_div"]/div[3]/h3[4]')
```
Comment (sshveta) [inline cfme/base/ui.py]: Can the locator be more specific ?

Comment (ganeshhubale) [reply cfme/base/ui.py]: @sshveta 
I have already tried to make it more specific than existing but I did not find more specific ```xpath``` than this one. Please let me know if you have any solution around it.

Comment (sshveta) [reply cfme/base/ui.py]: :+1:

Comment (izapolsk) [reply cfme/base/ui.py]: @ganeshhubale, isn't there any better xpath ?  
What I don't like in it
1. it uses indexes. So, any slight change of html page may break it.
2. Why do you use full xpath ? As far as I see your view is based on other views. So, I assume you can use relative locator here.

**Comment by izapolsk** [comment]
@ganeshhubale, @mshriver, are you sure this BZ has too be automated at all ?

**Comment by ganeshhubale** [comment]
@izapolsk 
I agree; it is static. But I am automating because of the BZ around it. 
It would be better to have test case around it. 
Please let me know your views @izapolsk @mshriver whether we are automating it or not.

**Comment by mshriver** [comment]
@ganeshhubale it does seem a bit frivolous to have a test case for this, when it looks more like a statement that should be in the `View.is_displayed` method. 

Is adding this title check to the page's `is_displayed` method not a sufficient validation that the title is correct now? Ultimately, this BZ is then tested by any CRUD test hitting this page.

**Comment by ganeshhubale** [comment]
@mshriver 
I am agree with you. I checked the ```is_displayed``` of view - ```AutomateImportExportView``` does not have this statement check.
I will update this is_displayed and  execute test which uses that view under this PR.
Thanks @mshriver @izapolsk

**Comment by dajoRH** [comment]
Lint report for commit b1947b0c7f2e8b65bc7a8430cb83e974d6325c7d:

`cfme/base/ui.py`:
- :bangbang: Line 1727:17: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/base/ui.py [hunk 2]
```diff
@@ -1709,12 +1709,22 @@ class import_git(View):     # noqa
...
     })
+    reset_title = Text(
+        ".//div[contains(@class, 'import-or-export')]/h3[contains(text(), 'Reset all')]"
+    )
 
     @property
     def is_displayed(self):
-        return self.in_import_export and self.export_all.is_displayed
+        # TODO: Remove check if the fix for BZ 1686054 is back ported to 5.9z
+        if self.appliance.version < "5.10":
```
Comment (mshriver) [inline cfme/base/ui.py]: PRT highlights the bug here - you're in a View class, which doesn't have an appliance. You can use `self.browser.product_version`

**Pull request merged**